google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -233,11 +239,72 @@ public boolean exists(String tableId, String rowKey) {
    * }</pre>
    *
    * @throws com.google.api.gax.rpc.ApiException when a serverside error occurs
+   * @deprecated Please use {@link BigtableDataClient#exists(TargetId, ByteString)} instead. Create
+   *     a TargetId for a table with {@link TableId#of(String)}.
    */
+  @Deprecated
   public boolean exists(String tableId, ByteString rowKey) {
     return ApiExceptions.callAndTranslateApiException(existsAsync(tableId, rowKey));
   }
 
+  /**
+   * Confirms synchronously if given row key exists or not on the specified {@link TargetId}.
+   *
+   * <p>Sample code:
+   *
+   * <pre>{@code
+   * try (BigtableDataClient bigtableDataClient = BigtableDataClient.create("[PROJECT]", "[INSTANCE]")) {
+   *   String tableId = "[TABLE]";
+   *   String key = "key";
+   *   String authorizedViewId = "[AUTHORIZED_VIEW]";
+   *
+   *   // Check if the row exists on the authorizedViewId
+   *   boolean isRowPresent = bigtableDataClient.exists(AuthorizedViewId.of(tableId, authorizedViewId), key);
```

Comment:
Here and everywhere else. I think the default sample code should use regular tables since they require less work to setup and get up and running faster